### PR TITLE
[MeshingApplication] Extending normalization methods in ComputeHessianSolMetricProcess

### PR DESCRIPTION
**Pull request by loumalouomega**
Adding value and gradient norm normalization methods as detailed in P.J. Frey, F. Alauzet; Anisotropic mesh adaptation for CFD computations; Comput. Methods Appl. Mech. Engrg. 194 (2005) 5068–5082

### CODE REVIEW

File: applications/MeshingApplication/python_scripts/mmg_process.py [hunk 1]
```diff
@@ -706,6 +726,20 @@ def __generate_double_list_from_input(self,param):
 
       return double_list
 
+    def __generate_string_list_from_input(self,param):
```
Comment (philbucher) [inline applications/MeshingApplication/python_scripts/mmg_process.py:729]: better use `GetStringArray`

Comment (loumalouomega) [reply applications/MeshingApplication/python_scripts/mmg_process.py:729]: It is an auxiliar method, should I update the rest of methods? (for int and double)

Comment (philbucher) [reply applications/MeshingApplication/python_scripts/mmg_process.py:729]: hm not sure
there is already `GetVector` for int and double

Up to you, but this method does exactly the same as GetStringArray

**Review by philbucher — APPROVED** [review]

File: applications/MeshingApplication/python_scripts/mmg_process.py (hunk 1 above)
Comment (loumalouomega) [reply applications/MeshingApplication/python_scripts/mmg_process.py:729]: True..., i will let like it is, and replace it in a mini PR

**Pull request merged**